src/App.js

```diff
@@ -1,30 +1,49 @@
-import React, { useState } from 'react';
-import { AppContext } from './context/AppContext';
+import React, { useState, useEffect } from 'react';
 
-import { toRN } from './helpers/transform';
 import './assets/css/global.css';
-import Header from './components/Header';
-import Title from './components/Title';
-import Transform from './components/Transform';
+
+import Header from './components/header/Header';
+import Transform from "./components/transform/Transform";
+import Title from "./components/title/Title";
+import { toJSS, toRN } from "./helpers/transform";
+
+const initialInputCss = `font-size: 18px;\nline-height: 24px;\ncolor: red;`;
+
+export const Context = React.createContext(null);
 
 function App() {
-  const rawSnippet = "font-size: 18px;\nline-height: 24px;\ncolor: red;";
-  const initialState = toRN(rawSnippet);
-  const [state, setState] = useState(initialState);
-  const [type, setType] = useState("React Native");
-  const [currentSnippet, setSnippet] = useState(rawSnippet);
-
-  return (
-    <AppContext.Provider
-      value={{ state, setState, rawSnippet, currentSnippet, setSnippet, type, setType }}
-    >
-      <div className="wrapper">
-        <Header />
-        <Title />
-        <Transform />
-      </div>
-    </AppContext.Provider>
-  );
+    const [outputType, setOutputType] = useState('JSS');
```

Comment:
Let's go with `React Native`.

Reply:
I made `React Native` the default option.